packages/header/src/Header.tsx

```diff
@@ -50,28 +52,32 @@ export const Header = ({
 					badgeLabel="beta"
 					background="bodyAlt"
 				/>
-				<MainNav
-					id={mainNavId}
-					activePath={activePath}
-					items={authenticated ? authenticatedLinks : []}
-					secondaryItems={
-						authenticated
-							? [
-									{
-										label: 'Sign out',
-										onClick: handleSignOut,
-										endElement: <AvatarIcon />,
-									},
-							  ]
-							: [
-									{
-										label: 'Sign in',
-										onClick: handleSignIn,
-										endElement: <AvatarIcon />,
-									},
-							  ]
-					}
-				/>
+				{!focusMode ? (
+					<MainNav
+						id={mainNavId}
+						activePath={activePath}
+						items={navItems}
+						secondaryItems={
+							authenticated
+								? [
+										{
+											label: 'Sign out',
```

Comment:
we want this label to alternate between 

"My account" when logged in, and
"Sign in" when logged out